gplately/download.py

```diff
@@ -444,13 +461,92 @@ def get_rasters(self, raster_id_string=None, time=None):
+            and with the requested filetype. 
+
+        Notes
+        -----
+        By default, get_netcdf_rasters will attempt to download age grids for each Ma timestep. For
+        example, Muller et al. 2019 has 251 rasters for 0-250Ma. If the `time` parameter is passed, 
+        only the raster for that timestep is returned. Otherwise, if `time` is not provided, full 
+        paths to all agegrids from the plate model will be returned. If `filetype` is not provided, 
+        age grids in netCDF format will be returned.
+        """
+
+        database = {
+
+            "Muller2019_netCDF" : ["https://www.earthbyte.org/webdav/ftp/Data_Collections/Muller_etal_2019_Tectonics/Muller_etal_2019_Agegrids/Muller_etal_2019_Tectonics_v2.0_netCDF.zip"],
+            "Muller2019_jpeg" : ["https://www.earthbyte.org/webdav/ftp/Data_Collections/Muller_etal_2019_Tectonics/Muller_etal_2019_Agegrids/Muller_etal_2019_Tectonics_v2.0_jpgs.zip"],
+            "Muller2019_png" : ["https://www.earthbyte.org/webdav/ftp/Data_Collections/Muller_etal_2019_Tectonics/Muller_etal_2019_Agegrids/Muller_etal_2019_Tectonics_v2.0_pngs.zip"],
+            "Muller2016_netCDF" : ["https://www.earthbyte.org/webdav/ftp/Data_Collections/Muller_etal_2016_AREPS/Muller_etal_2016_AREPS_Agegrids/Muller_etal_2016_AREPS_Agegrids_v1.17/Muller_etal_2016_AREPS_v1.17_netCDF.zip"],
+            "Muller2016_jpeg" : ["https://www.earthbyte.org/webdav/ftp/Data_Collections/Muller_etal_2016_AREPS/Muller_etal_2016_AREPS_Agegrids/Muller_etal_2016_AREPS_Agegrids_v1.17/Muller_etal_2016_AREPS_v1.17_jpgs.zip"],
+            "Muller2016_png" : ["https://www.earthbyte.org/webdav/ftp/Data_Collections/Muller_etal_2016_AREPS/Muller_etal_2016_AREPS_Agegrids/Muller_etal_2016_AREPS_Agegrids_v1.17/Muller_etal_2016_AREPS_v1.17_pngs.zip"],
+        }
+
+        archive_formats = tuple([".gz", ".xz", ".bz2"])
+        # Set to true if we find the given collection in database
+        found_collection = False
+        raster_filenames = []
+        for collection, zip_url in database.items():
+            # Isolate the plate model and the file type
+            plate_model = collection.split("_")[0]
+            raster_type = collection.split("_")[-1]
+            if (self.file_collection.lower() == plate_model.lower()
+                and filetype.lower() == raster_type.lower()
+                ):
+                found_collection = True
+                raster_filenames = _order_filenames_by_time(
+                    _collect_file_extension(
+                        _fetch_from_web(zip_url[0]), _match_filetype_to_extension(filetype))
+                )
+                
+                if time is not None:
+                    raster_filenames = _order_filenames_by_time(raster_filenames)[time]
+                break
+
+        if found_collection is False:
+            raise ValueError("%s not in collection database." % (raster_id_string))
+        return raster_filenames
```

Comment:
It is not useful to return the filename. Read it in using the `Raster` object first, then return a (masked) numpy array:

```python
raster = gplately.Raster(filename=muller_2019_age_grid, extent=[-180,180,-90,90])
return raster.data
```
I'm not sure how this passed me by for the `get_topology_geometries` method, but you should not return filenames to the user. Instead, for this function, you should return pygplates Features or FeatureCollections.